pkg/crd/kubermatic/v1/cluster.go

```diff
@@ -61,6 +61,11 @@ const (
 	MaxEtcdClusterSize     = 9
 )
 
+const (
+	AnnotationCNIPluginKind    = "cni-plugin.k8c.io/kind"
```

Comment:
I may be missing something, but I do not see where these annotations are being used

Reply:
Good catch :-) I started using the annotations then I thought that extending the cluster CRD was cleaner.

Reply:
yeah I thought so :)